sdk/core/Azure.Core/samples/DynamicContent.md

```diff
@@ -0,0 +1,73 @@
+# Azure SDK Dynamic JSON samples
+
+Azure SDK client [Protocol methods](ProtocolMethods.md) do not take or return model types.  JSON response content can be accessed using BCL JSON types, but use of these APIs can result in code that is difficult to read and obscures the author's intent.  To improve the developer experience, Azure.Core provides a dynamic layer over JSON APIs.
+
+## Accessing Response Content
+
+### Get `dynamic` from response
+
+Dynamic content is obtained from the `Response` return value.
+
+```C# Snippet:GetDynamicJson
+Response response = await client.GetWidgetAsync("123");
+dynamic widget = response.Content.ToDynamic();
+```
+
+### Get a JSON property
+
+JSON properties are read using dynamic member access.
+
+```C# Snippet:GetDynamicJsonProperty
+Response response = await client.GetWidgetAsync("123");
+dynamic widget = response.Content.ToDynamic();
+string name = widget.Name;
+```
+
+### Check whether an optional property is present
+
+Optional properties are checked for null.
+
+[TODO: implement](https://github.com/Azure/azure-sdk-for-net/issues/33856)
+
+### Collections can be enumerated
+
+Dynamic JSON arrays are `IEnumerable` and can be iterated over with the `foreach` keyword.
+
+[TODO: Add sample]
+
+## Setting Request Content
+
+Authoring JSON from scratch to pass as inputs to protocol methods is [done using anonymous types](ProtocolMethods.md#2-create-and-send-a-request).  When working with Azure services, however, it is common to retrieve a value from from the service, make some changes to it, and send the updated value back to the service.  This is called a "round-trip scenario."
```

Comment:
Does it need to be done via anonymous types, or is that just the recommendation?

Reply:
It is the officially recommended guidance, but it's possible to author it any way that works with [RequestContent.Create()](https://learn.microsoft.com/en-us/dotnet/api/azure.core.requestcontent.create?view=azure-dotnet).  I can reword this to make that clear - thanks!